nuxeo/templates/deployment.yaml

```diff
@@ -32,8 +32,13 @@ spec:
 {{- with .Values.podLabels }}
 {{ toYaml . | indent 8 }}
 {{- end }}
-{{- if .Values.podAnnotations }}
       annotations:
+        checksum/configmap: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+        checksum/secret: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
+        rollme: {{ ternary (randAlphaNum 5 | quote) .Values.image.tag (or (eq "latest" (lower .Values.image.tag)) (hasSuffix "snapshot" (lower .Values.image.tag))) }}
```

Comment:
`rollme` annotations ensure if the docker images version is a mutable(`latest` or `-SNAPSHOT` version), it triggers deployment